include/zephyr/bluetooth/conn.h

```diff
@@ -991,6 +991,18 @@ struct bt_conn_cb {
  */
 void bt_conn_cb_register(struct bt_conn_cb *cb);
 
+/** @brief Unregister all connection callbacks.
+ *
+ *  Unregister callbacks to cleanup connections.
+ */
+void bt_conn_cb_unregister(void);
```

Comment:
I think it should be <code>bt_conn_cb_unregister<b>_all</b></code>. The plain `bt_conn_cb_unregister` must be reserved for a future `bt_conn_cb_unregister(struct bt_conn_cb *cb)`, that is the symmetrical API to `bt_conn_cb_register`.